src/main/webapp/src/app/app.module.ts

```diff
@@ -18,24 +18,32 @@ import {NgModule} from '@angular/core';
 import {AppRoutingModule} from './app-routing.module';
 import {MaterialComponents} from './common/material-components';
 import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {MatCardModule} from '@angular/material/card';
+import {CarouselModule} from 'ngx-owl-carousel-o';
```

Comment:
I have a question about the carousel: Is this carousel the only one available for angular or does angular have its own carousel?

Reply:
In my research I didn't see a material carousel component. You either had to hard code it yourself,use ng bootstrap, or use a different package. https://material.angular.io/components/categories